### ISSUE TITLE
EPG Not working

### ISSUE THREAD

**Issue by pcwgr1** [free-greek-iptv/greek-iptv#34]
EPG has stopped working   source not found.  Any updates

**Comment by komhsgr** [comment]
https://github.com/GreekTVApp/EPG-GRCY/releases/download/EPG/epg.xml.gz

i check working fine

**Comment by pcwgr1** [comment]
Thank you I solved it.

Efharisto poli kai poli kali douleia

From: komhsgr ***@***.***>
Reply-To: free-greek-iptv/greek-iptv ***@***.***>
Date: Monday, April 5, 2021 at 11:34 AM
To: free-greek-iptv/greek-iptv ***@***.***>
Cc: ***@***.***" ***@***.***>, Author ***@***.***>
Subject: Re: [free-greek-iptv/greek-iptv] EPG Not working (#34)


https://github.com/GreekTVApp/EPG-GRCY/releases/download/EPG/epg.xml.gz

i check working fine

**Comment by stav8124** [comment]
I uploaded your Greek channels as a M3U file to Smart TV and they work great thank you,  Is there a way getting the EPG to work for Smart TV?

**Comment by komhsgr** [comment]
yes working https://github.com/GreekTVApp/epg-greece-cyprus/releases/download/EPG/epg.xml
